contracts/0.8.25/utils/AccessControlConfirmable.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2025 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+// See contracts/COMPILERS.md
+pragma solidity 0.8.25;
+
+import {AccessControlEnumerable} from "@openzeppelin/contracts-v5.2/access/extensions/AccessControlEnumerable.sol";
+import {Confirmations} from "./Confirmations.sol";
+
+/**
+ * @title AccessControlConfirmable
+ * @author Lido
+ * @notice An extension of AccessControlEnumerable that allows executing functions by mutual confirmation.
+ * @dev This contract extends Confirmations and AccessControlEnumerable and adds a confirmation mechanism.
+ */
+abstract contract AccessControlConfirmable is AccessControlEnumerable, Confirmations {
+
+    constructor() {
+        __Confirmations_init();
```

Comment:
It seems this call is unnecessary, as the contract will be used behind a proxy and initialization will occur in the implementation. This also makes it asymmetric with `Confirmable2Addresses`, which doesn’t call init in its constructor